picard/const/__init__.py

```diff
@@ -186,7 +186,6 @@
     "%title%"
 
 
-DEFAULT_NUMBERED_SCRIPT_NAME = N_("My script %d")
 DEFAULT_SCRIPT_NAME = N_("My script")
 DEFAULT_COVER_IMAGE_FILENAME = "cover"
 DEFAULT_NUMBERED_PROFILE_NAME = N_("My profile %d")
```

Comment:
Perhaps we can also use `unique_numbered_title()` for profiles?

Reply:
Excellent idea.  I wish I had thought of it.  I'll have a look.